model/MailboxHandler.php

```diff
@@ -43,6 +43,11 @@ protected function initStruct() {
             # TODO: add virtual 'notified' column and allow to display who received a vacation response?
         );
 
+        if (Config::read('forgotten_user_password_reset')) {
+            $this->struct['phone'] = pacol(   1,          1,      0,      'text', 'pCreate_mailbox_phone'         , 'pCreate_mailbox_phone_desc'       , '');
+            $this->struct['email_other'] = pacol(   1,          1,      0,      'mail', 'pCreate_mailbox_email'         , 'pCreate_mailbox_email_desc'       , '');
+        }
+
```

Comment:
This has been fixed: the additional fields for password reset are always added to the structure, but are displayed in the form only when the password reset feature is enabled.